hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/reconfig/TestOmReconfigure.java

```diff
@@ -97,4 +98,23 @@ public void testOmAdminUsersReconfigure() throws Exception {
         ozoneManager.getOmAdminUsernames().contains(userB));
   }
 
+  /**
+   * Test reconfigure om "ozone.readonly.administrators"
```

Comment:
Checkstyle complains about missing period:

```
hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/reconfig/TestOmReconfigure.java
 101: First sentence should end with a period.
```

```suggestion
   * Test reconfigure om "ozone.readonly.administrators".
```